test/service/test_jobs.py

```diff
@@ -258,7 +258,7 @@ def test_logs_creation_failure_during_algorithm_retrieval(self):
         ], logstream.getvalue().splitlines())
 
     def test_logs_creation_failure_during_scene_retrieval(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.